scm-dao-xml/src/main/java/sonia/scm/store/JAXBConfigurationEntryStore.java

```diff
@@ -186,42 +191,44 @@ private void store() {
     LOG.debug("store configuration to {}", file);
 
     context.withMarshaller(m -> {
-      m.setProperty(Marshaller.JAXB_FRAGMENT, Boolean.TRUE);
+        m.setProperty(Marshaller.JAXB_FRAGMENT, Boolean.TRUE);
 
       CopyOnWrite.withTemporaryFile(
         temp -> {
-          try (IndentXMLStreamWriter writer = XmlStreams.createWriter(temp)) {
-            writer.writeStartDocument();
+          try (Writer ioWriter = Files.newBufferedWriter(temp, StandardCharsets.UTF_8)) {
+            try (IndentXMLStreamWriter writer = XmlStreams.createWriter(ioWriter)) {
```

Comment:
This could be done in one `try` block, avoiding the second pair of parens.
```suggestion
          try (Writer ioWriter = Files.newBufferedWriter(temp, StandardCharsets.UTF_8);
               IndentXMLStreamWriter writer = XmlStreams.createWriter(ioWriter)) {
```